.github/workflows/action.yml

```diff
@@ -2,15 +2,15 @@ name: build
 on: [push, pull_request]
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
I was a bit too optimistic of just changing the CI and expecting it to just work.
I would need some help from your previous experience to make the build work.
I wonder why `ppa:rabbitmq/rabbitmq-erlang` did not work in the end?